Integrations/integration-Pwned.yml

```diff
@@ -160,4 +161,24 @@ script:
       description: Compromised Domains
       related: ""
     description: Check if domain has been compromised
+  - name: email
```

Comment:
why change the name from `pwned-email` to `email`? I think our conventions are to command names must start with brand name